arwenmandos/exec.go

```diff
@@ -101,6 +103,17 @@ func (ae *ArwenTestExecutor) GetVM() vmi.VMExecutionHandler {
 
 // GetVMHost returns de vm Context from the vm context map
 func (ae *ArwenTestExecutor) GetVMHost() arwen.VMHost {
+	skipStackLevels := 2
```

Comment:
this is some hardcoded stuff. Can't we do better ?

Reply:
@andrei-marinica

Reply:
I don't really know what that variable means. This is the mandos executor, but I guess we could extract to a constant.

Reply:
Deleted, this is unnecessary. This code appears to have been copied from the old `GetVMHost()`, when it had VM hook logs. But here, `arwenmandos.GetVMHost()` was not called from within VM hooks, so the whole logging code here did nothing useful.